qaframework-bdd-tests/src/test/resources/features/refapp-2.x/16-PhoneNumberValidation/phoneNumberValidation.feature

```diff
@@ -0,0 +1,23 @@
+Feature:Phone Number validation
+
+  Background:
+    Given user goes to active visit search page
+   
+  @selenium
+  @phoneNumberValidation
+  Scenario: Phone Number validation
+    When user searches for patient using patient identifier
+    Then system loads patient dashboard page of last active visit
+    And user clicks on show contact link
+    And user clicks on edit contact info link
+    And user clicks on phone number button
+    And user clears phone number
+    And user enters new phone number using script element
+    And user clicks on confirm edit button
+    Then system returns a message validator
+    And user clears the phone number input area
+    And user enters the correct phone number
+    And user clicks again on confirm edit button
+    And user clicks on confirm patient button
+    And user re clicks on show contact link
```

Comment:
is this a typo? `re clicks`

Reply:
Its not a typo. i did that because i wanted to have two different scenarios, remember am using these two scenarios at the same so if they remain looking the same, selenium with throw duplicate exceptions.

Reply:
what is the meaning of `re clicks?`

Reply:
It simply means that with in this workflow the user clicks the same button twice.

Reply:
> It simply means that with in this workflow the user clicks the same button twice.

do you have a documentation that gives the meaning of `re clicks`?

Reply:
Uhmm but do you understand why am using re clicks in the first place. i have  two scenarios  of which the first scenario says. `User clicks on show contact link` [here ](https://github.com/openmrs/openmrs-contrib-qaframework/blob/1487d3c4f21e3230f0d476aadbb97e94bed4fd46/qaframework-bdd-tests/src/test/resources/features/refapp-2.x/16-PhoneNumberValidation/phoneNumberValidation.feature#L11). then again am supposed to click on show contact link for the second round. i cannot directly use same english(readable and making meaning) like User clicks on show contact link` . if i do this selenium will throw duplicate exceptions.

Reply:
Let me simply the readability is probably bringing confusion.

Reply:
Does that make sense.